post-scan-actions/aws-python-promote-or-quarantine/terraform/iam.tf

```diff
@@ -0,0 +1,68 @@
+# Lambda execution policy for ScanningBucketNamePolicy
+resource "aws_iam_policy" "scanning_bucket_name_policy" {
+  name_prefix = "ScanningBucketNamePolicy-"
+  description  = "AWS IAM Policy for managing aws lambda role"
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:GetObject",
+        "s3:DeleteObject",
+        "s3:GetObjectTagging"
+      ],
+      "Resource": "arn:aws:s3:::${var.ScanningBucketName}/*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:ListBucket"
+      ],
+      "Resource": "arn:aws:s3:::${var.ScanningBucketName}"
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:PutObject",
+        "s3:PutObjectTagging",
+        "s3:PutObjectAcl"
+      ],
+      "Resource": [
+        "arn:aws:s3:::${var.PromoteBucketName}/*",
```

Comment:
Please use the same condition from the original policy. 
https://github.com/trendmicro/cloudone-filestorage-plugins/blob/0c1e219478b3e8c2ff6aaf762a65847c0c3a9457/post-scan-actions/aws-python-promote-or-quarantine/template.yml#L122

Reply:
As far I know, these conditions only works on CF, if they actually work on terraform I will be happy to add them here. Could you confirm?